maintainers/team-list.nix

```diff
@@ -87,6 +87,13 @@ with lib.maintainers; {
     scope = "Maintain GNOME desktop environment and platform.";
   };
 
+  home-assistant = {
+    members = [
+      hexa
+    ];
```

Comment:
```suggestion
      hexa
      fab
    ];
```

Reply:
Uff, that will get messy with the suggestions.

Reply:
Merged into one.

Reply:
Thanks